packages/empty-state/src/BasicEmptyState/BasicEmptyState.tsx

```diff
@@ -32,22 +32,24 @@ export const BasicEmptyState = forwardRef(
   ) => {
     const { theme, darkMode } = useDarkMode(darkModeProp);
 
-    if (!isComponentType(primaryButton, 'Button')) {
-      console.error(
-        'The `primaryButton` prop in BasicEmptyState should be of type LeafyGreen Button.',
-      );
-    }
+    if (primaryButton || secondaryButton) {
+      if (!isComponentType(primaryButton, 'Button')) {
+        console.error(
+          'The `primaryButton` prop in BasicEmptyState should be of type LeafyGreen Button.',
+        );
+      }
 
-    if (!isComponentType(secondaryButton, 'Button')) {
-      console.error(
-        'The `secondaryButton` prop in BasicEmptyState should be of type LeafyGreen Button.',
-      );
-    }
+      if (!isComponentType(secondaryButton, 'Button')) {
```

Comment:
Would this still log an error if only a primaryButton was passed in?

Reply:
<img width="1778" alt="Screenshot 2023-05-17 at 2 52 43 PM" src="https://github.com/mongodb/leafygreen-ui/assets/5740159/5644b945-348e-4cb1-bcb4-95bb91a2e736">

Reply:
think I fixed it, rfal